pkg/plugins/networkinterface/apinet/apinet.go

```diff
@@ -180,17 +163,12 @@ func (p *Plugin) Apply(ctx context.Context, spec *api.NetworkInterfaceSpec, mach
 		return nil, fmt.Errorf("error waiting for nic to become ready: %w", err)
 	}
 
-	// Fetch the updated object to get the ID or any other updated fields
-	if err := p.host.APINetClient().Get(ctx, apinetNicKey, apinetNic); err != nil {
-		return nil, fmt.Errorf("error fetching updated apinet network interface: %w", err)
-	}
-
-	log.V(1).Info("Host device is ready", "HostDevice", hostDev)
-	nicIPs := make([]net.IP, 0, len(apinetNic.Spec.IPs))
+	nicIPs := make([]string, 0, len(apinetNic.Spec.IPs))
 	for _, apinetNicIP := range apinetNic.Spec.IPs {
 		// TODO: do proper IP type conversion here
-		nicIPs = append(nicIPs, net.ParseIP(apinetNicIP.String()))
+		nicIPs = append(nicIPs, apinetNicIP.String())
 	}
+	log.V(1).Info("Host device is ready", "HostDevice", hostDev, "IPs", nicIPs)
```

Comment:
are you think it is good idea log whole structure hostDev and maybe IPs too, how it can help ops guys? Or maybe different level as info, if you need it.